crypto/armor_test.go

```diff
@@ -25,7 +25,7 @@ import (
 	"github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1"
 	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
 	"github.com/cosmos/cosmos-sdk/crypto/xsalsa20symmetric"
-	"github.com/cosmos/cosmos-sdk/runtime"
+	_ "github.com/cosmos/cosmos-sdk/runtime"
```

Comment:
dumb question, why do we still blank import runtime ? is it because of depinject ?

Reply:
Right, [this test](https://github.com/cosmos/cosmos-sdk/blob/82bd39bac55ce5a6ab2e07fcc3442de33752abaf/crypto/armor_test.go#L83-L90) uses `depinject` to initialize, so without runtime imported the Provider funcs are absent.

To be honest it seems a bit not golang, I'd like to see a composition root where Providers are defined rather than importing packages for side effects.